### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-CPD Parent: 4 image thumbnails labels

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1958]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Parent: 
https://staging.oregondigital.org/concern/generics/df727084j?locale=en

The 4 thumbnails have the following failed instance for labels:
label: Ensures every form element has a label

Path:
#batch_document_df727080f

Snippet:
`<input type="checkbox" onfocus="setGalleryCheckbox(this)" onblur="setGalleryCheckbox(this)" onchange="setGalleryCheckbox(this)" name="batch_document_ids[]" id="batch_document_df727080f" value="df727080f" class="batch_document_selector">`

### Suggested Fix

Fix ONE of the following:

- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by CGillen** [comment]
Fixed earlier

**Comment by azle** [comment]
@CGillen , I revalidated the page and received the following 2 errors below.

<img width="1000" alt="Screen Shot 2022-01-31 at 10 01 44 AM" src="https://user-images.githubusercontent.com/8353860/151847962-f2826b7b-bb91-46b9-a7a7-e3d5f0488183.png">


<img width="1020" alt="Screen Shot 2022-01-31 at 10 02 27 AM" src="https://user-images.githubusercontent.com/8353860/151848066-c780fe28-6b91-4b79-966e-39d5a23fd22b.png">

**Comment by CGillen** [comment]
Above issue is related to an UV issue we currently can not fix. Please disregard until we can edit the UV iframe (if we decide to)
